api/app/models/true_or_false_question.rb

```diff
@@ -0,0 +1,3 @@
+class TrueOrFalseQuestion < ApplicationRecord
+  belongs_to :course
+end
```

Comment:
the reason why your tests are failing is because there are no validations here

Reply:
Done